lang/py/src/avro/codecs.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python
```

Comment:
It isn't necessary to keep this else condition around anymore.

We may also wish to be careful about the term "codecs". Like `avro.io`, `avro.codecs` is likely to collide with [a core Python module](https://docs.python.org/2.7/library/codecs.html). While namespaces ought to prevent any serious conflict, we will probably end up using Python codecs in avro, so we should avoid any confusion here. Mainly, what I'm concerned about is that Python's definition of `codec` is different from Avro's. Avro seems to mean "compression algorithm", whereas Python's is more generically for transforming data. Maybe we should add a docstring here to make it really clear.